src/components/grocery-view/grocery-section/CategoryRow.js

```diff
@@ -1,22 +1,24 @@
 // import PropTypes from 'prop-types';
-// import React from 'react';
+import React from 'react';
 
-// // Components
-// import GroceryItem from './GroceryItem';
+// Components
+import { Grid } from '@material-ui/core';
+import GroceryItem from './GroceryItem';
 
-// // Styles
-// import styles from '../styles/category-row.module.css';
+// Styles
+import styles from '../styles/category-row.module.css';
 
-// const CategoryRow = () => (
-//     <div>
-//       CategoryRow
-//     </div>
-// );
+const CategoryRow = () => {
+  const groceryItems = [1, 2, 3, 4, 5, 6].map((index) => (
+    <Grid item>
+      <GroceryItem key={index} />
+    </Grid>
```

Comment:
Nice. But might be good to go through and leave //REFACTOR comments in the parts where we are using fake data like this.
In this example we'll eventually link it with GraphQL and stuff. It might help when we have to do things later.